README.md

```diff
@@ -647,7 +706,117 @@ jmeter:
 | endpoint |  | string | Yes |
 | thresholdMillis |  | int64 | Yes |
 
+
+### Junit
+
+The check take's pod spec as input complete its job and post the result of the generated junit result files
```

Comment:
This is unclear to me. I missed it the first time.

Reply:
It takes pod spec i.e. container and other info and waits for it to complete its job. Once complete parses the generated junit reports and post back the result.
Can you suggest what should be the phrasing for this?